### DISCUSSION TITLE
Multivarite irregular sampled Time Series Forecasting,When will it be supported?

### DISCUSSION THREAD

**Discussion by luoolu** [autogluon/autogluon#3567] [Q&A]
Looking forward to supporting this feature.

**Comment by shchur** [comment]
Thank you for the suggestion. We have added functionality for automatically imputing irregularly-sampled time series (#3454), it will become available in the next release. We currently do not have plans for supporting multivariate forecasting and focus on forecasting multiple univariate time series (panel data).

**Comment by luoolu** [comment]
Thanks, Looking forward to the next release!
